### DISCUSSION TITLE
Embargoed Jurisdictions

### DISCUSSION THREAD

**Discussion by adlai** [community/community#23176] [New to GitHub]
<p>I noticed in a recent post that <a href="https://github.community/t/iranians-are-now-free-to-access-github-thank-you-but-i-want-the-remainder-of-my-premium-account-back/154053">one of the embargos</a> has been lifted.</p>
<p>Is there a list of the remaining ones, if any?</p>

**Comment by adlai** [comment]
<p>Although this is obviously a question theoretically more appropriately addressed to the legal department of the parent corporation, it is also both a question whose theoretical answers have zero practical relevance, yet also one whose practical answer is painfully difficult to obtain independently; thus, I refloat the original question, lest it drown unanswered.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi -</p>
<p>This is from our FAQ on Trade Controls:</p>
<blockquote>
<p>Crimea, Cuba, Iran, North Korea, and Syria. With respect to Iran, however, GitHub now has a license from the U.S. Treasury Department’s Office of Foreign Assets Control (OFAC) to provide cloud services to developers located or otherwise resident in that country.</p>
</blockquote>
<p>You can find further information here:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/github/site-policy/github-and-trade-controls#frequently-asked-questions" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/site-policy/github-and-trade-controls#frequently-asked-questions" target="_blank" rel="noopener">GitHub and Trade Controls - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/site-policy/github-and-trade-controls#frequently-asked-questions" target="_blank" rel="noopener">//docs.github.com/en/github/site-policy/github-and-trade-controls</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I hope that helps!</p>

**Comment by adlai** [comment]
<aside class="quote group-github-employees quote-modified" data-username="canuckjacq" data-post="3" data-topic="154402">
<div class="title">
<div class="quote-controls"></div>
 canuckjacq:</div>
<blockquote>
<p>from our FAQ on Trade Controls:</p>
<blockquote>
<p>Crimea, Cuba, Iran, North Korea, and Syria. With respect to Iran, however, GitHub now has a license from the U.S. Treasury Department’s Office of Foreign Assets Control (OFAC) to provide cloud services to developers located or otherwise resident in that country.</p>
</blockquote>
</blockquote>
</aside>
<p>Thank you. I suppose that the provision of cloud services is still moderated, and anything written by foreigners that falls foul of export controls may become silently inaccessible to those within any of those locations.</p>
